scripts/scenario_20_training1.lua

```diff
@@ -135,6 +84,7 @@ There will be not much resistance, so you can try different approaches and tacti
 
 Here's your chance to beat up some helpless opponents.
 Commander Saberhagen out.]]))
+    globalMessage("Test")
```

Comment:
"Test"?

Reply:
Oops - Sorry that was a debugging remnant, I just removed it.